packages/lesswrong/components/form-components/SelectLocalgroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+import MenuItem from '@material-ui/core/MenuItem';
+import { useMulti } from '../../lib/crud/withMulti';
+
+/* A form input for selecting a localgroup -
```

Comment:
```suggestion
/**
 * A form input for selecting a localgroup -
```

Good habit: use TS-Doc comments when documenting. (Unfortunately, I believe the way we register components breaks TS-Doc for components in particular.)